lifecycle_msgs/srv/GetState.srv

```diff
@@ -0,0 +1,3 @@
+string node_name
+---
+uint8 current_state
```

Comment:
What about putting the constants for the default life cycle state in a separate file that gets included by both message definitions?